scripts/updatecli/updatecli_acs72.yml

```
@@ -0,0 +1,31 @@
+acs:
+  version: "7.2"
+
+adw:
+  version: "2.9"
+
+aos:
+  version: "1.4"
+
+apiExplorer:
+  version: "7.2"
+
+dsync:
+  version: "3.6"
+
+googleDrive:
+  version: "3.2"
+
+search:
+  version: "2.0"
+
+searchEnterprise:
+  version: "3.1"
+
+transform:
+  version: "2.5"
+
+trouter:
+  version: "1.5"
```

as trouter & transform versions have been bumped respectively to `2.x` & `3.x` I would use

```suggestion
transform:
  version: "2"

trouter:
  version: "1"
```